controllers/quickstart_constants.go

```diff
@@ -238,6 +238,54 @@ spec:
         failedTaskHelp: This task isn’t verified yet. Try the task again.
       summary:
         success: You have successfully created BucketClass
+        failed: Try the steps again. 
+    - title: Encrypt your StorageClass
+      description: |-
+
+          StorageClass encryption enables you to provide individual keys to each of your StorageClusters.
+
+
+          You can create an encrypted StorageClass while configuring your StorageCluster, or when creating a new StorageClass.
+
+
+          **To create an encrypted StorageClass while setting up your cluster:**
+
+          1. After installing OpenShift Data Foundation, select **Create StorageSystem**.
+
+          2. In the creation wizard, go to **Security and network** step.
+
+          3. Check the **StorageClass encryption** box.
+
+          4. Select your **Key management service provider**.
+
+          5. Provide your connection details.
+
+
+          **To create a new encrypted StorageClass:**
+
+          1. In the navigation menu, select **Storage > StorageClasses**, and select **Create StorageClass**.
+
+          2. In the **Provisioner** field, select **openshift-storage.rbd.csi.ceph.com**.
+
+          3. Check the **Enable Encryption** box.
+
+          4. Select your **Key management service provider**.
+
+          5. Provide your connection details.
+      review:
+        instructions: |-
+          ####  To verify that the StorageClass is encrypted:
+          1. Go to **Storage > StorageClasses**.
+          If you encrypted the StorageClass while setting up your cluster, check that a new StorageClass with the prefix/suffix **encrypted** is created.
+
+          2. Select the newly created StorageClass and select the **YAML** tab.
+
+          Does the value of the parameter **encrypted** equal to **true**?
+
+          Is the value of the parameter **encryptionKMSID** equal to the KMS connection's **Service name**?
```

Comment:
```suggestion
          Is the value of the parameter **encryptionKMSID** equal to the KMS connection's service name?
```
No need to bold generic terms unless it is a field name, in which case use the proper field name

Reply:
`Service name` is the proper field name that we show in the UI.

Reply:
but we are in the YAML tab i presume after step 2 ?
`Select the newly created StorageClass and select the **YAML** tab.`
in which case we can use the field name or keep it as `service name` itself

Reply:
ack... I will change it to 
```suggestion
          Is the value of the parameter **encryptionKMSID** equal to the KMS connection's service name?
```